reproman/interface/execute.py

```diff
@@ -228,12 +250,22 @@ def post_command(self):
             sort_packages=False,
             find_inputs_outputs=True)
 
+        # include extra trace output if we have any
```

Comment:
This comment doesn't really add anything to the code.  IMO what would be useful is a comment (somewhere, probably not here) about the purpose of the extra trace file.